.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -37,3 +37,6 @@ Closes #
 - [ ] I have added tests that prove my fix is effective or that my feature works
 - [ ] If I included new strings, I have used `trans.` to make them localizable.
       For more information see our [translations guide](https://napari.org/developers/translations.html).
+
+<!--Feel free to message us in Zulip for any questions or remind us to review:
```

Comment:
```suggestion
<!--Feel free to message us in Zulip for any questions or remind us to review if there is no reaction in 4 days:
```

Maybe something like this? To point that there is expected to wait few days?

Reply:
Yes maybe a guide on how long to wait would be useful.

Though I think I will close this in favour of the bot suggested by @Carreau. Are you supportive of such a bot: https://github.com/napari/napari/pull/6091#issuecomment-1649511666 ?

Reply:
Just waiting for any further objections

Reply:
I like boot too.